src/screens/ENSSearchSheet.js

```diff
@@ -140,14 +139,7 @@ export default function ENSSearchSheet() {
               </Text>
             </Inline>
           )}
-          {isIdle && (
```

Comment:
Ideally, we would like to keep this way of doing it. `PendingRegistrations` may be used elsewhere, so having a divider in there might not make sense.
Why doesn't this way work?

Reply:
if we want to do this we need to pass the pending registrations state to this component to check if there's any since the designs say that we don't want the divider if the are no pending registrations, so it doesn't depend on the search. For me it make more sense to have the divider inside since it depends on the state, maybe we can pass a flag to the component to render the divider or not, WDYT about that @jxom ?

Reply:
ah right, let's leave it for now then